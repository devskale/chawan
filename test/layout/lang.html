<!DOCTYPE html>
<html lang=red>
<style>
:lang(red) { color: red }
:lang(green) { color: green }
:lang(blue) { color: blue }
body { color: yellow !important }
div { color: purple }
</style>
<div>
test
<div lang=green>test</div>
<div lang=blue>test</div>
<div lang>test</div>
