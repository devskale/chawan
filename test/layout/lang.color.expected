[38;2;255;40;40mtest[39m
[38;2;41;169;42mtest[39m
[38;2;88;87;255mtest[39m
[38;2;191;64;191mtest[39m
